docs/data_management/res/cheaha_project_directory.csv

```diff
@@ -0,0 +1,10 @@
+Permissions/Responsibilities,Research Computing,Members,Manager,PI
+Create a project directory,Upon PI request,,,
+Request a project directory,,,,Yes ✔️
+Add/remove member to/from the project space,Upon PI approval/Manager request,,,
+Grant access to users to a specific folder,Upon PI approval/Manager request,,Yes ✔️,Yes ✔️
```

Comment:
We will want to clarify what are RC's responsibilities and what are RC's services upon request.

- Granting access to a specific folder is an RC service, not an RC responsibility.
  - If a folder is "locked", meaning, no group members can change the permissions nor gain access to it. Then they need us to fix this.
  - If they need assistance configuring or reconfiguring permissions. This is provided as a convenience.
- Add/remove members is a joint responsibility. The supervisor is responsible for who belongs to the group. RC is responsible for executing changes upon request.
- Creating project directory is a joint responsibility. Supervisor is responsible for making the request and for the created allocation. RC is responsible for executing the request.